src/datasources/asset-calibration/AssetCalibrationDataSource.ts

```diff
@@ -41,11 +43,30 @@ export class AssetCalibrationDataSource extends DataSourceBase<AssetCalibrationQ
 
   systemAliasCache: TTLCache<string, SystemMetadata> = new TTLCache<string, SystemMetadata>({ ttl: metadataCacheTTL });
 
+  private readonly assetComputedDataFields = new Map<AssetCalibrationFieldNames, ExpressionTransformFunction>([
+    [
+      AssetCalibrationFieldNames.LOCATION,
+      (value: string, operation: string, options?: TTLCache<string, unknown>) => {
```

Comment:
ExpressionTransformFunction?

Reply:
Yes, this would be the implementation for it. Would you like to have the params as a separate type?